### ISSUE TITLE
Support latest redmine view add-maintenance-message-to-bodytop

### ISSUE THREAD

**Issue by jdehaan** [tools-aoeur/redmine_maintenance_mode#4]
```log
E, [2022-10-12T00:05:50.133619 #1873] ERROR -- : Deface: 'add-maintenance-message-to-login-3.3' matched 0 times with 'table'
E, [2022-10-12T00:05:50.137928 #1873] ERROR -- : Deface: [ERROR] The original source for 'add-maintenance-message-to-login-3.4' has changed, this override should be reviewed to ensure it's still valid.
E, [2022-10-12T00:05:50.170194 #1873] ERROR -- : Deface: [ERROR] The original source for 'add-maintenance-message-to-bodytop' has changed, this override should be reviewed to ensure it's still valid.
E, [2022-10-12T00:05:53.168666 #1874] ERROR -- : Deface: [ERROR] The original source for 'add-maintenance-message-to-bodytop' has changed, this override should be reviewed to ensure it's still valid.
```

**Comment by jdehaan** [comment]
Better completely remote the deface stuff and use redmine hooks instead:

* `view_account_login_top` for above the login
* `view_layouts_base_body_top` for the top of the base layout

**Comment by tools-aoeur** [comment]
Closed by #5
